maven/bnd-export-plugin/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>bnd-export-plugin</name>
```

Comment:
The other maven plugins follow the naming pattern: bnd-XXX-maven-plugin. This one one should also. I expect it would be bnd-export-maven-plugin.